src/audio.c

```diff
@@ -1940,6 +1953,11 @@ int audio_send_digit(struct audio *a, char key)
 			return EINVAL;
 		}
 
+		err = dtmf_queue_digit(&a->tx.dtmfgen, key, a->tx.ac->crate, 0, 0);
```

Comment:
ac->srate should be used here, instead of ac->crate.

does it support both 1 and 2 channels ?